tools/android/packaging/Makefile.in

```diff
@@ -102,7 +102,7 @@ python: | xbmc/assets
 	cd xbmc/assets/python2.7/lib/python2.7/; rm -rf test config lib-dynload
 
 res:
-	mkdir -p xbmc/res xbmc/res/raw xbmc/res/values images
+	mkdir -p xbmc/gen/org/xbmc/kodi xbmc/res xbmc/res/raw xbmc/res/values images
```

Comment:
its doesn't hurt either. We use it everywhere, so it should be the same here, if only for the sake of consistency.